### PR TITLE
sys-power/tlp: new package, add 1.5.0

### PR DESCRIPTION
**Pull request by jayofdoom**
This is a direct import from ::guru, with the only changes being to metadata.xml to propose myself as proxy maintainer. 

The original person listed as the guru maintainer, Duje Mihanović, indicated to me in an email they'd like to remain a maintainer as this moves into ::gentoo, so I set them as a proxied maintainer in metadata.xml as well.

### CODE REVIEW

**Comment by jayofdoom** [comment]
I don't understand why the bot doesn't think I did a GCO... maybe the amended commit confused it?

File: sys-power/tlp/tlp-1.5.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+inherit udev optfeature
+
+DESCRIPTION="Optimize laptop battery life"
+HOMEPAGE="https://linrunner.de/tlp/"
+SRC_URI="https://github.com/linrunner/TLP/archive/refs/tags/${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/TLP-${PV}"
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64"
+RESTRICT="mirror"
```
Comment (Flowdalic) [inline sys-power/tlp/tlp-1.5.0.ebuild:15]: I guess this restriction could be lifted if in ::gentoo?

File: sys-power/tlp/tlp-1.5.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+# It's uncertain if elogind/systemd is actually required, however, without the sleep
+# hooks working, which require one of them, it doesn't seem like this app is very useful.
+RDEPEND="virtual/udev
+		|| ( sys-auth/elogind sys-apps/systemd )"
+DEPEND="${RDEPEND}"
```
Comment (Flowdalic) [inline sys-power/tlp/tlp-1.5.0.ebuild:20]: Are those really DEPENDs?

File: sys-power/tlp/tlp-1.5.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+HOMEPAGE="https://linrunner.de/tlp/"
+SRC_URI="https://github.com/linrunner/TLP/archive/refs/tags/${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/TLP-${PV}"
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64"
+RESTRICT="mirror"
+# It's uncertain if elogind/systemd is actually required, however, without the sleep
+# hooks working, which require one of them, it doesn't seem like this app is very useful.
+RDEPEND="virtual/udev
```
Comment (Flowdalic) [inline sys-power/tlp/tlp-1.5.0.ebuild:18]: TLP seems to have a few perl scripts too, so maybe dev-lang/perl should be added as RDEPEND?

File: sys-power/tlp/tlp-1.5.0.ebuild [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+EAPI=8
+
+inherit udev optfeature
+
+DESCRIPTION="Optimize laptop battery life"
+HOMEPAGE="https://linrunner.de/tlp/"
+SRC_URI="https://github.com/linrunner/TLP/archive/refs/tags/${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/TLP-${PV}"
+LICENSE="GPL-2"
```
Comment (Flowdalic) [inline sys-power/tlp/tlp-1.5.0.ebuild:12]:
```suggestion

LICENSE="GPL-2"
```

Add a space between DESCRIPTION< HOMEPAGE, SRC_URI, S and LICENSE. See skel.ebuild.

File: sys-power/tlp/tlp-1.5.0.ebuild [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+SLOT="0"
+KEYWORDS="~amd64"
+RESTRICT="mirror"
+# It's uncertain if elogind/systemd is actually required, however, without the sleep
```
Comment (Flowdalic) [inline sys-power/tlp/tlp-1.5.0.ebuild:16]:
```suggestion

# It's uncertain if elogind/systemd is actually required, however, without the sleep
```

Al;so space between RESTRICT and RDEPEND

File: sys-power/tlp/tlp-1.5.0.ebuild (hunk 2 above)
Comment (jayofdoom) [reply sys-power/tlp/tlp-1.5.0.ebuild:20]: This doesn't link against these packages, just needs them for runtime so I'm only putting them as DEPEND.

**Pull request closed without merging**